src/api/app/views/webui/shared/bs_requests/_requests_filter.html.haml

```diff
@@ -26,6 +26,25 @@
                                                               value: state,
                                                               checked: selected_filter[:state]&.include?(state.to_s) }
 
+  .mt-2.mb-2.accordion-item.border-0
+    .px-3.py-2.accordion-button.no-style{ data: { 'bs-toggle': 'collapse', 'bs-target': '#request-filter-stag-proj' },
+                                                       aria: { expanded: 'true', controls: 'request-filter-stag-proj' } }
+      %b Staging  Project
+      .selected-content.small.ms-1
+    .px-4.pb-2.accordion-collapse.collapse.show#request-filter-stag-proj
+      .dropdown.form-multi-select{ 'data-name': 'staging projects' }
+        %button.form-select.text-start#filter-staging-projects-button{ type: 'button', data: { 'bs-toggle': 'dropdown' } }
+          Select staging project
+        .dropdown-menu.w-100#stag-proj-dropdown
+          %input.form-control#stag-proj-search{ placeholder: 'Filter staging project' }
+          - Project.where.not(staging_workflow_id: nil).each do |sp|
+            .dropdown-item-text
+              - checked = selected_filter[:stag_proj]&.include?(sp.name) || defined?(project) && project == sp
+              = render partial: 'webui/shared/check_box', locals: { label: sp.name,
+                                                                    key: "stag_proj[#{sp.id}]", name: 'stag_proj[]',
+                                                                    value: sp.name,
+                                                                    checked: checked }
+
```

Comment:
```suggestion
    .px-4.pb-2.accordion-collapse.collapse.show#request-filter-stag-proj
      %input.form-control#stag-proj-search{ placeholder: 'Filter staging project' }
      .dropdown-item-text
        - Project.where.not(staging_workflow_id: nil).each do |sp|
          - checked = selected_filter[:stag_proj]&.include?(sp.name) || defined?(project) && project == sp
          = render partial: 'webui/shared/check_box', locals: { label: sp.name,
                                                                key: "stag_proj[#{sp.id}]", name: 'stag_proj[]',
                                                                value: sp.name,
                                                                checked: checked }

```

Now that filters are collapsible at their top level, I think we should drop the `dropdown-menu` structure and show directly the filter and the values instead.

**Before**
![image](https://github.com/user-attachments/assets/ad5d4950-1225-49ff-b923-24b3bcbe2d6e)


**After**
![image](https://github.com/user-attachments/assets/de5caa25-8aa6-4b7a-9cd7-b15635763e48)



Keep in mind the `js` needs to be adapted after this dropping because it relies on `#stag-proj-dropdown` `.closest()` elements which are now different.

Reply:
Nice. Let me look into it

Reply:
@ncounter, I don't think removing the dropdown is a good idea. The reason is that there are more than 300 staging projects in production, and without the dropdown, all of them will be visible in the filter section.

Reply:
The plan is to use https://github.com/openSUSE/open-build-service/pull/17218 afterwards

Reply:
As discussed offline, we'll tackle this point within https://github.com/openSUSE/open-build-service/pull/17218 eventually.